src/routes/driver/driverRoutes.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import { deleteRideOffer } from '../../controllers/driver/deleteOffer';
+import { editRideOffer } from '../../controllers/driver/editOffer';
+import { editProfile } from '../../controllers/driver/editProfile';
+import { addRide, createdOffer } from '../../controllers/driver/rideOffer';
+
+import { createDriverAccount } from '../../controllers/driver/signUp';
+import {
+  isLoggedIn,
+  validateDriverLoginDetails,
+  validateExistingDriver,
+  validateSignup,
+} from '../../middleware';
+
+const driverRouter = express.Router();
+
+driverRouter.post(
+  '/driver/signup',
+  validateSignup,
+  validateExistingDriver,
+  createDriverAccount
+);
+driverRouter.post('/driver/login', validateDriverLoginDetails);
+driverRouter.post('/driver/create', isLoggedIn, addRide);
+driverRouter.delete(
+  '/driver/delete-ride-offer/:offerId',
```

Comment:
The HTTP method name already has `delete`, there is no need to add it on the route again. it could be something like `/offer/:offerId`